Database/Create_timeseriesmetadata_table.sql

```diff
@@ -0,0 +1,18 @@
+create table `sdliticadb`.`timeseriesmetadata` (
```

Comment:
One more thing I found is that when table is created with the name specified here it is not recognized by code and I'm getting Table not found error. Code expects to see table this name `TimeSeriesMetadata` (camecased)